api/envoy/extensions/matching/common_inputs/stats/v3/stats.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package envoy.type.matcher.v3;
```

Comment:
This needs to change to `envoy.extensions.matching.common_inputs.stats.v3`.

Reply:
DONE.